pyproject.toml

```diff
@@ -26,7 +26,8 @@ dependencies = [
     "rasterio",
     "geopandas",
     "shapely",
-    "cvxopt"
+    "cvxopt",
```

Comment:
You need to get rid of `cvxopt` here. Since it is still listed as a dependency, we still try to install it when we install topotoolbox.